fe/fe-core/src/main/java/org/apache/doris/mysql/privilege/UserProperty.java

```diff
@@ -274,6 +274,12 @@ public void update(List<Pair<String, String>> properties) throws UserException {
                 if (keyArr.length != 1) {
                     throw new DdlException(PROP_SQL_BLOCK_RULES + " format error");
                 }
+                // check if sql_block_rule has already exist
+                for (String rule_name : value.replaceAll(" ","").split(",")){
```

Comment:
```suggestion
                for (String ruleName : value.replaceAll(" ","").split(",")){
```

Reply:
Ok, thanks for your suggestion, I will do it latter.